prototype/snippet/jsonnet/snippet.go

```diff
@@ -315,7 +314,7 @@ func replace(jsonnet string, imports []ast.Import) string {
 	})
 
 	for _, im := range imports {
-		param := paramReplacementPrefix + strings.TrimPrefix(im.File.Value, paramPrefix) + paramReplacementSuffix
+		param := paramReplacementPrefix + strings.TrimPrefix(im.File.Value, paramPrefix)
```

Comment:
If I'm reading this code correctly, this actually obviates the need for us to return a `snippet.Template` at all. Why not just return a string? Or, perhaps make a second function that emits a string, and keep this one?